src/Android/NativeSensorListener.cpp

```diff
@@ -179,6 +179,25 @@ Java_org_xcsoar_NativeSensorListener_onI2CbaroSensor(JNIEnv *env, jobject obj,
                            AtmosphericPressure::Pascal(pressure));
 }
 
+gcc_visibility_default
+JNIEXPORT void JNICALL
+Java_org_xcsoar_NativeSensorListener_onEngineSensboxSensors(JNIEnv *env,
```

Comment:
No such Java method exists

Reply:
What Java files?
Did you understand my comment? That your PR does not make sense that way?

Reply:
> Did you understand my comment?

Obviously I did not. I get no error when compiling so the compiler seems not to care about the Java method. Can you give me some more context?

Reply:
This piece of C++ code implements the native part of a Java method. But the Java method you're implementing doesn't exist, therefore you cannot implement it.
The C++ compiler doesn't know about Java methods, therefore it has no reason to complain. But I complain. Not because your code is syntactically wrong, but because it's logically wrong, it makes no sense.

Reply:
So to make sens, I should add not the Java files, I should add a Java file...
Thanks